src/ucs/profile/profile.c

```diff
@@ -29,22 +30,43 @@ typedef struct ucs_profile_global_context {
     unsigned                      num_locations; /**< Number of valid locations */
     unsigned                      max_locations; /**< Size of locations array */
     pthread_mutex_t               mutex;         /**< Protects updating the locations array */
-    ucs_time_t                    start_time;    /**< Thread context init time */
-    ucs_time_t                    end_time;      /**< Thread end time */
+
+    pthread_key_t                 tls_key;       /**< TLS key for per-thread context */
+    ucs_list_link_t               thread_list;   /**< List of all thread contexts */
+
```

Comment:
minor: excessive empty line? here and at the beginning of the structure definition

Reply:
removed few empty lines